### PR TITLE
Add tall crafting status

### PR DESCRIPTION
**Pull request by Laiff**
Added option to show tall version of crafting status 

@divran 
- [x]  100% tall crafting status screen
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11349

### CODE REVIEW

File: src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java [hunk 1]
```diff
@@ -252,7 +257,7 @@ public void drawScreen(final int mouseX, final int mouseY, final float btn) {
         final int offY = 23;
         int y = 0;
         int x = 0;
-        for (int z = 0; z <= 4 * 5; z++) {
+        for (int z = 0; z <= 4 * rows; z++) {
```
Comment (repo-alt) [inline src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java:260]: shouln't it be (rows-1) or <= changed to < ?

**Comment by Divran** [comment]
Excellent

File: src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java (hunk 1 above)
Comment (Laiff) [reply src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java]: You right.. partially :) There are to many magic numbers around. Make this relevant to current rendering.

File: src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java [hunk 2]
```diff
@@ -249,10 +255,10 @@ public void drawScreen(final int mouseX, final int mouseY, final float btn) {
 
         this.tooltip = -1;
 
-        final int offY = 23;
+        final int offY = SECTION_HEIGHT;
         int y = 0;
         int x = 0;
-        for (int z = 0; z <= 4 * 5; z++) {
+        for (int z = 0; z <= ITEMS_PER_ROW * rows; z++) {
```
Comment (repo-alt) [inline src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java:261]: perhaps magic 2 in line 274 shouldbe changed to (ITEMS_PER_ROW-1) then

Comment (Laiff) [reply src/main/java/appeng/client/gui/implementations/GuiCraftingCPU.java:261]: reworked. There are some other bugs was present here

**Review by repo-alt — APPROVED** [review]

**Pull request merged**